### PR TITLE
ci: ignore protoc version comment on go generate

### PR DESCRIPTION
**Pull request by galargh**
Go generate leaves protoc version comments in some generated files. This PR ensures that we ignore it in the go check workflow.

### CODE REVIEW

File: .github/workflows/go-check.yml [hunk 1]
```diff
@@ -15,4 +15,6 @@ concurrency:
 
 jobs:
   go-check:
-    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
+    uses: pl-strflt/uci/.github/workflows/go-check.yml@release-v0.0.13
```
Comment (galargh) [inline .github/workflows/go-check.yml:18]: This is just testing before release. We should restore v0.0 once v0.0.13 is out.

```suggestion
    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
```

**Review by sukunrt — APPROVED** [review]

**Comment by sukunrt** [comment]
Thanks @galargh

File: .github/workflows/go-check.yml (hunk 1 above)
Comment (sukunrt) [reply .github/workflows/go-check.yml:18]: So should we merge this as is or wait for the release?

Comment (galargh) [reply .github/workflows/go-check.yml:18]: We can go back to this now:
```suggestion
    uses: pl-strflt/uci/.github/workflows/go-check.yml@v0.0
```

**Review by marten-seemann — APPROVED** [review]

**Pull request merged**